documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -42,7 +42,7 @@ For example, when you change service account labels or annotations using the `te
 [id='ref-operator-use-strimzi-pod-sets-feature-gate-{context}']
 === UseStrimziPodSets feature gate
 
-The `UseStrimziPodSets` feature gate introduced the `StrimziPodSet` custom resource for managing Kafka and ZooKeeper pods, replacing the use of Kubernetes `StatefulSet` resources.
+The `UseStrimziPodSets` feature gate introduced the `StrimziPodSet` custom resource for managing Kafka pods, replacing the use of Kubernetes `StatefulSet` resources.
```

Comment:
I would keep this as it was. This is basically just a historical record about what the `UseStrimziPodSets` feature gate was doing. Maybe we should think about how long we keep the old feature gates in the docs.

Reply:
Right. Updated
We could change the list to _Recently graduated feature gates_ and keep those 
that have _important_ upgrade/downgrade information. But maybe we could drop UnidirectionalTopicOperator feature gate and ServiceAccountPatching feature gate now.

Reply:
Maybe UseKRaft feature gate too